src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            System.out.println(lottery.getRandomBall());
```

Comment:
Consider defining the number of balls to generate as a constant. Magic numbers like '3' directly in the code can be confusing and are not easily maintainable. Instead, you can define a constant at the beginning of your class with a descriptive name.